containers/css/badges.css

```diff
@@ -0,0 +1,55 @@
+:root{
+    --inactive:red;
+    --active:green;
+    --neutral:darkgray;
+    --primary-color:white;
+}.badges-container{
+    margin: 1rem;
+}
+.per-badge{
+    margin: 1rem ;
+    display: flex;
+}
+.badge{
+    margin: 1rem;
+    height: 0.7rem;
+    width: 0.7rem;
+    border-radius: 50%;
+    background-color: var(--neutral);
+    border: 0.1rem solid var(--primary-color);
+}
+.active-badge{
+    background-color: var(--active);
+}
+.inactive-badge{
+    background-color: var(--inactive);
+}
+
+
+.badge-on-icon{
+    position: relative;
+    font-size:xx-large ;
+}
+.badge-on-icon .badge{
+    text-align: center;
+    color: white;
+    font-weight: bold;
+    height: 0.9rem;
+    width: 0.9rem;
+    margin: 0rem;
```

Comment:
Try to use a standardized scale system like system of 8px so that design looks consistent throughout.